core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -510,6 +512,12 @@ public SystemSessionProperties(
                         "Enable compression in exchanges",
                         featuresConfig.isExchangeCompressionEnabled(),
                         false),
+                enumProperty(
+                        COMPRESSION_KIND,
+                        "Compression kind in exchanges and page serializations",
```

Comment:
"Compression codec used for data in exchanges and spills"

Reply:
Actually I think that we should have 2 separate configuration properties: one for exchanges (data transmission), one for spills (data storage)

Reply:
> Actually I think that we should have 2 separate configuration properties: one for exchanges (data transmission), one for spills (data storage)

Yes i haved added a session property and a configuraion for exchanges, and a configuration for spills~ The `FileSingleSpillerFactory` doesn't have a session property api